primitives/rpc/src/lib.rs

```diff
@@ -39,6 +40,28 @@ pub struct TransactionStatus {
 	pub logs_bloom: Bloom,
 }
 
+pub trait EvmRuntimeAddressMapping {
```

Comment:
Yeah I thought about using `AddressMapping<Vec<u8>>` but since there's an impl `impl<T: From<H160>>` already in causes ambiguous implementation since `From<H160>` can implemented for `Vec<u8>` as well. It could be solved by adding the fn `into_account_id_bytes` to the trait `AddressMapping` and may be reused while computing the `account_id`


```rust
impl<H: Hasher<Out = H256>> AddressMapping<AccountId32> for HashedAddressMapping<H> {
	fn into_account_id(address: H160) -> AccountId32 {
		AccountId32::from_slice(&Self::into_account_id_bytes(address));
	}

	fn into_account_id_bytes(address: H160) -> Vec<u8> {
		let mut data = [0u8; 24];
		data[0..4].copy_from_slice(b"evm:");
		data[4..24].copy_from_slice(&address[..]);
		let hash = H::hash(&data);

		Into::<[u8; 32]>::into(hash).to_owned()

	}
}
```

thoughts?

Reply:
Seems in the pallet-evm `Config` we pass `AccountId` to `AddressMapping` which doesn't have the `AsRef<[u8]>` trait bound